include/TPP/Dialect/Xsmm/XsmmUtils.h

```diff
@@ -77,6 +104,9 @@ FailureOr<UnaryFlags> getUnaryFlags(Type inputType, Type outputType);
 
 // Compute the broadcasting flags for 'operandType' based on 'outputType'.
 enum class OperandPos { LHS = 0, RHS = 1 };
+FailureOr<BinaryFlags> getBinFlags(ArrayRef<int64_t> shapeOutput,
```

Comment:
Transpose is unary, no addition on Binary is needed.

Reply:
Same comment as above, will remove.